Resources/Prototypes/_NF/Recipes/Lathes/storage.yml

```
@@ -1,192 +1,148 @@
   id: WeaponCaseShortExplosives
+  parent: NFBaseToolboxRecipe
   result: WeaponCaseShortExplosives
   name: lathe-recipe-WeaponCaseShortExplosives-name
-  category: StorageNF
-  completetime: 3
-  materials:
-    Steel: 300
+
+# Cardboard boxes
 
 - type: latheRecipe
   id: NFBoxMREEmpty
+  parent: NFBaseBoxRecipe
   result: BoxMREEmpty
   name: lathe-recipe-NFBoxMREEmpty-name
-  category: StorageNF
-  completetime: 1
-  applyMaterialDiscount: false
-  materials:
-    Cardboard: 100
 
 - type: latheRecipe
   id: NFHappyHonkEmpty
+  parent: NFBaseBoxRecipe
   result: HappyHonkEmpty
   name: lathe-recipe-NFHappyHonkEmpty-name
-  category: StorageNF
-  completetime: 1
-  applyMaterialDiscount: false
-  materials:
-    Cardboard: 100
 
 - type: latheRecipe
   id: NFHappyHonkMimeEmpty
+  parent: NFBaseBoxRecipe
   result: HappyHonkMimeEmpty
   name: lathe-recipe-NFHappyHonkMimeEmpty-name
-  category: StorageNF
-  completetime: 1
-  applyMaterialDiscount: false
-  materials:
-    Cardboard: 100
 
 - type: latheRecipe
   id: NFHappyHonkNukieEmpty
+  parent: NFBaseBoxRecipe
   result: HappyHonkNukieEmpty
   name: lathe-recipe-NFHappyHonkNukieEmpty-name
-  category: StorageNF
-  completetime: 1
-  applyMaterialDiscount: false
-  materials:
-    Cardboard: 100
 
 - type: latheRecipe
   id: NFHappyHonkMcCargoEmpty
+  parent: NFBaseBoxRecipe
   result: HappyHonkMcCargoEmpty
   name: lathe-recipe-NFHappyHonkMcCargoEmpty-name
-  category: StorageNF
-  completetime: 1
-  applyMaterialDiscount: false
-  materials:
-    Cardboard: 100
 
 - type: latheRecipe
   id: NFHappyHonkCluwneEmpty
+  parent: NFBaseBoxRecipe
   result: HappyHonkCluwneEmpty
   name: lathe-recipe-NFHappyHonkCluwneEmpty-name
-  category: StorageNF
-  completetime: 1
-  applyMaterialDiscount: false
-  materials:
-    Cardboard: 100
 
 - type: latheRecipe
   id: NFFoodBoxPizza
+  parent: NFBaseBoxRecipe
   result: FoodBoxPizza
   name: lathe-recipe-NFFoodBoxPizza-name
-  category: StorageNF
-  completetime: 1
-  applyMaterialDiscount: false
-  materials:
-    Cardboard: 100
 
 - type: latheRecipe
   id: NFFoodBoxDonutEmpty
+  parent: NFBaseBoxRecipe
   result: FoodBoxDonutEmpty
   name: lathe-recipe-NFFoodBoxDonutEmpty-name
-  category: StorageNF
-  completetime: 1
-  applyMaterialDiscount: false
-  materials:
-    Cardboard: 100
 
 - type: latheRecipe
   id: NFFoodBoxNuggetEmpty
+  parent: NFBaseBoxRecipe
   result: FoodBoxNuggetEmpty
   name: lathe-recipe-NFFoodBoxNuggetEmpty-name
-  category: StorageNF
-  completetime: 1
-  applyMaterialDiscount: false
-  materials:
-    Cardboard: 100
 
 - type: latheRecipe
   id: NFFoodContainerEggEmpty
+  parent: NFBaseBoxRecipe
   result: FoodContainerEggEmpty
   name: lathe-recipe-NFFoodContainerEggEmpty-name
-  category: StorageNF
-  completetime: 1
-  applyMaterialDiscount: false
-  materials:
-    Cardboard: 100
 
 - type: latheRecipe
   id: NFFoodBoxSushi
+  parent: NFBaseBoxRecipe
   result: FoodBoxSushi
   name: lathe-recipe-NFFoodBoxSushi-name
```

Think this one should keep the plastic.
```suggestion
  name: lathe-recipe-NFFoodBoxSushi-name
  materials:
    Plastic: 100
```